.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        target: [test]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup CI Environment
+        uses: yetanalytics/actions/setup-env@v0.0.4
```

Comment:
You can use this action instead: https://github.com/yetanalytics/action-setup-env

Reply:
done